<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkpMainWindow</class>
 <widget class="QMainWindow" name="SkpMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkpMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="SkpTitleWidget" name="titleWidget" native="true"/>
    </item>
    <item>
     <widget class="SkpTabBar" name="tabBar" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkpTitleWidget</class>
   <extends>QWidget</extends>
   <header>skp_title_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SkpTabBar</class>
   <extends>QWidget</extends>
   <header>skp_tabBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
